File: problems/985/solution.go
package main
func sumEvenAfterQueries(A []int, queries [][]int) []int {
    ans := make([]int, len(A))
            
    for i, query := range queries {
        A[query[1]] += query[0]
        sum := 0
        
        for _, v := range A {
            if v % 2 == 0 {
                sum += v
            }
        }
        
        ans[i] = sum
    }
    
    return ans
}

File: problems/985/solution.js
/**
 * @param {number[]} A
 * @param {number[][]} queries
 * @return {number[]}
 */
var sumEvenAfterQueries = function(A, queries) {
    let ans = [];
    
    for (let i = 0; i < queries.length; i++) {
        let query = queries[i];
        
        A[query[1]] += query[0];
        ans[i] = A.filter(v => v % 2 == 0).reduce((acc, cv) => acc + cv, 0);
    }
    
    return ans;
};